include/api.php

```diff
@@ -2340,25 +2345,33 @@ function api_format_messages($item, $recipient, $sender)
 {
 	// standard meta information
 	$ret = [
-			'id'                    => $item['id'],
-			'sender_id'             => $sender['id'] ,
-			'text'                  => "",
-			'recipient_id'          => $recipient['id'],
-			'created_at'            => api_date($item['created']),
-			'sender_screen_name'    => $sender['screen_name'],
-			'recipient_screen_name' => $recipient['screen_name'],
-			'sender'                => $sender,
-			'recipient'             => $recipient,
-			'title'                 => "",
-			'friendica_seen'        => $item['seen'],
-			'friendica_parent_uri'  => $item['parent-uri'],
+		'id'                    => $item['id'],
+		'sender_id'             => $sender['id'] ,
+		'text'                  => "",
+		'recipient_id'          => $recipient['id'],
+		'created_at'            => api_date(defaults($item, 'created', DateTimeFormat::utcNow())),
```

Comment:
This field is always filled. So we should add it to the test data. But a check here is superfluous.

Reply:
Most item fields are having some "NOT NULL" check, additionally the Item::insert function does guarantee that these fields are always set. We also fetch the item array here with some function from the item class.

There should be no way that ```$item['created']``` isn't filled or does not exist in the regular environment.

And: If this here raises a warning then it is a clear indicator that something went wrong and that we have to fix it. When we are working like above, we will not recognise possible bugs and will have harder times finding them. In this case it is an indicator that our test data isn't complete.